rts/System/Sound/OpenAL/Sound.cpp

```diff
@@ -507,7 +507,14 @@ void CSound::OpenLoopbackDevice(const std::string& deviceName)
 	desiredSpec.samples = 4096;
 	desiredSpec.callback = RenderSDLSamples;
 	desiredSpec.userdata = this;
-
+	
+	/* SDL bug: can return devices with >2 channels (3D surround), even if we ask for just 2.
+ 	* (if tested) This causes the 2 "primary" channels to be moved in the 3D space compared to their "normal" state.
+ 	* (if not tested) We are afraid that this risks the 2 "primary" channels to be moved in the 3D space compared to their "normal" state, but this remains to be tested.
```

Comment:
if tested / not tested was supposed to reflect what our actual state of knowledge is. if you tested then remove the "if not tested" line, or vice versa

Reply:
changed and comitted